### PR TITLE
codespell: fix remaining issues

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

Those just keep popping up.

### CODE REVIEW

**Comment by fabian18** [comment]
I checked out this PR and ran  `./dist/tools/codespell/check.sh`.

```
drivers/at24cxxx/at24cxxx.c:150: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:152: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:155: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:156: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:157: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:174: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:175: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:177: clen ==> clean, clan
drivers/at24cxxx/at24cxxx.c:178: clen ==> clean, clan
drivers/at86rf215/at86rf215_o-qpsk.c:205: octects ==> octets
drivers/include/at30tse75x.h:122: contiously ==> continuously
drivers/include/at86rf2xx.h:88: seetings ==> settings
drivers/include/l3gxxxx.h:26: Initializaton ==> Initialization
drivers/include/l3gxxxx.h:376: Initializaton ==> Initialization
drivers/include/l3gxxxx.h:1304: ore ==> or
drivers/include/periph/spi.h:327: truct ==> struct
drivers/include/xbee.h:137: onw ==> own
drivers/lsm303dlhc/lsm303dlhc.c:159: endiness ==> endianness
drivers/netdev/ieee802154.c:257: conter ==> counter, conteur
drivers/sht3x/sht3x.c:118: inititalize ==> initialize
drivers/sx1280/include/sx1280_params.h:37: commun ==> commune, common
pkg/fatfs/fatfs_vfs/fatfs_vfs.c:405: RO ==> TO, ROW, ROB, ROD, ROE, ROT
pkg/libcose/contrib/libcose_riot_crypto.c:30: clen ==> clean, clan
pkg/libcose/contrib/libcose_riot_crypto.c:35: clen ==> clean, clan
pkg/libcose/contrib/libcose_riot_crypto.c:39: clen ==> clean, clan
pkg/libcose/contrib/libcose_riot_crypto.c:44: clen ==> clean, clan
pkg/libcose/contrib/libcose_riot_crypto.c:48: clen ==> clean, clan
pkg/lua/doc.txt:69: ovewritten ==> overwritten
pkg/lua/include/lua_run.h:19: scrips ==> scripts
pkg/tinyvcdiff/include/vcdiff_vfs.h:38: Maxmimum ==> Maximum
pkg/wolfssl/doc.txt:110: followint ==> following
release-notes.txt:8095: ro ==> to, row, rob, rod, roe, rot
release-notes.txt:9689: packes ==> packs, packed, packages
release-notes.txt:9982: packes ==> packs, packed, packages
release-notes.txt:10912: unnused ==> unused
sys/arduino/include/spiport.hpp:58: significat ==> significant
sys/hashes/md5.c:53: patter ==> pattern
sys/include/bloom.h:75: descreases ==> decreases
sys/include/net/gnrc/netif/conf.h:73: dequeing ==> dequeuing
sys/include/test_utils/result_output.h:18: perticular ==> particular
sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c:215: clen ==> clean, clan
sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c:217: clen ==> clean, clan
sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c:218: clen ==> clean, clan
sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c:753: clen ==> clean, clan
sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c:755: clen ==> clean, clan
sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c:756: clen ==> clean, clan
sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c:462: Verfiy ==> Verify
sys/net/network_layer/sixlowpan/sixlowpan_print.c:158: destinaton ==> destination
sys/net/network_layer/sixlowpan/sixlowpan_print.c:173: destinaton ==> destination
sys/net/network_layer/sixlowpan/sixlowpan_print.c:196: destinaton ==> destination
sys/net/network_layer/sixlowpan/sixlowpan_print.c:217: destinaton ==> destination
sys/posix/include/sys/socket.h:269: shal ==> shall
sys/posix/include/sys/socket.h:295: shal ==> shall
sys/posix/pthread/include/pthread_barrier.h:89: destoy ==> destroy
sys/puf_sram/puf_sram.c:27: conter ==> counter, conteur
tests/gnrc_sixlowpan/tests/01-run.py:26: destinaton ==> destination
tests/gnrc_tcp/tests-as-root/01-run.py:148: verfies ==> verifies
tests/gnrc_tcp/tests-as-root/01-run.py:192: verfies ==> verifies
tests/periph_backup_ram/README.md:6: incemented ==> incremented
tests/periph_gpio_ll/main.c:66: Deatils ==> Details
tests/pkg_fatfs_vfs/main.c:105: RO ==> TO, ROW, ROB, ROD, ROE, ROT
tests/pkg_fatfs_vfs/main.c:137: RO ==> TO, ROW, ROB, ROD, ROE, ROT
tests/pkg_fatfs_vfs/main.c:363: RO ==> TO, ROW, ROB, ROD, ROE, ROT
tests/pkg_lz4/main.c:39: nam ==> name
tests/pkg_lz4/tests/01-run.py:26: nam ==> name
tests/pkg_uzlib/main.c:45: nam ==> name
tests/unittests/tests-bloom/tests-bloom-sets.h:576: donot ==> do not, donut
```

Some are variable names. Some of them are remaining, right?

File: dist/tools/codespell/ignored_words.txt [hunk 1]
```diff
@@ -125,3 +125,15 @@ sur
...
 unsecure
+
+# nce (neighbor cache entry) => once, nice
+nce
+
+# warmup => warm up, warm-up
+warmup
+
+# clen => clean, clan
+clen
```
Comment (fabian18) [inline dist/tools/codespell/ignored_words.txt:136]: Do you have an idea why this was the only variable name that `codespell` is allergic to? I am very sure there are more variable names that are not an english word.

Comment (benpicco) [reply dist/tools/codespell/ignored_words.txt:136]: I have no idea - I'm also wondering why those start popping up all of the sudden.

**Review by fabian18 — APPROVED** [review]
fixes a lot of errors. ACK

**Pull request merged**